<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>51</width>
      <height>51</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>601</width>
      <height>389</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Anlam</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Saydamlık %</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>100</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="opacitySlider">
         <property name="minimum">
          <number>30</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="singleStep">
          <number>5</number>
         </property>
         <property name="pageStep">
          <number>5</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBothSides</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QListWidget" name="relatedList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>258</width>
         <height>330</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>393</width>
         <height>319</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">width:10px</string>
       </property>
       <property name="autoScroll">
        <bool>false</bool>
       </property>
       <property name="autoScrollMargin">
        <number>16</number>
       </property>
       <property name="tabKeyNavigation">
        <bool>true</bool>
       </property>
       <property name="movement">
        <enum>QListView::Static</enum>
       </property>
       <property name="resizeMode">
        <enum>QListView::Fixed</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Sözcükler</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTextBrowser" name="resultsBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>333</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>10</y>
      <width>551</width>
      <height>80</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Arama yapılacak kategori</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>10</y>
       <width>524</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButtonTeknikTerimler">
        <property name="text">
         <string>Teknik Terimler</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButtonTurkceIngilizce">
        <property name="text">
         <string>Türkçe-&gt;ingilizce</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pushButtonTipTerimleri">
        <property name="text">
         <string>Tıp Terimleri</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButtonBilgisayarTerimleri">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Bilgisayar Terimleri</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButtonIngilizceTurkce">
        <property name="text">
         <string>İngilizce-&gt;Türkçe</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButtonBilisimTerimleri">
        <property name="text">
         <string>Bilişim Terimleri</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButtonElektronikTerimleri">
        <property name="text">
         <string>Elektronik Terimleri</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>94</y>
      <width>551</width>
      <height>61</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>451</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="searchBox">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>Aramanızı buraya yazın, yapıştırın ya da sürükleyip bırakın</string>
        </property>
        <property name="statusTip">
         <string>Aramanızı buraya yazın, yapıştırın ya da sürükleyip bırakın</string>
        </property>
        <property name="whatsThis">
         <string/>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="dragEnabled">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string>Ara</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>Temizle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Yapıştır</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Kopyala</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QCheckBox" name="findIncludings">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>36</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>İçinde geçenleri de bul</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <zorder>frame_2</zorder>
   <zorder>frame</zorder>
   <zorder>pushButton</zorder>
   <zorder>gridLayoutWidget_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>656</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QMenu" name="menuG_r_n_m">
     <property name="title">
      <string>Görünüm</string>
     </property>
     <addaction name="actionWindows"/>
     <addaction name="actionPlastique"/>
     <addaction name="actionOxygen"/>
     <addaction name="actionCleanlooks"/>
     <addaction name="actionBespin"/>
     <addaction name="actionCDE"/>
     <addaction name="actionGTK"/>
     <addaction name="actionQtcurve"/>
     <addaction name="actionMotif"/>
    </widget>
    <addaction name="menuG_r_n_m"/>
    <addaction name="actionHide"/>
    <addaction name="actionExit"/>
    <addaction name="actionShow"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionWindows">
   <property name="text">
    <string>Windows</string>
   </property>
  </action>
  <action name="actionPlastique">
   <property name="text">
    <string>Plastique</string>
   </property>
  </action>
  <action name="actionOxygen">
   <property name="text">
    <string>Oxygen</string>
   </property>
  </action>
  <action name="actionCleanlooks">
   <property name="text">
    <string>Cleanlooks</string>
   </property>
  </action>
  <action name="actionBespin">
   <property name="text">
    <string>Bespin</string>
   </property>
  </action>
  <action name="actionCDE">
   <property name="text">
    <string>CDE</string>
   </property>
  </action>
  <action name="actionGTK">
   <property name="text">
    <string>GTK</string>
   </property>
  </action>
  <action name="actionQtcurve">
   <property name="text">
    <string>Qtcurve</string>
   </property>
  </action>
  <action name="actionMotif">
   <property name="text">
    <string>Motif</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Çıkış</string>
   </property>
  </action>
  <action name="actionHide">
   <property name="text">
    <string>Gizle</string>
   </property>
  </action>
  <action name="actionShow">
   <property name="text">
    <string>Göster</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>clearButton</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>searchBox</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>closeApplication()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>599</x>
     <y>447</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>relatedList</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>relatedSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>450</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>opacitySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>608</x>
     <y>515</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>515</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>searchBox</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>416</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>searchBox</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>504</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>searchBox</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>closeApplication()</slot>
  <slot>relatedSelected()</slot>
  <slot>setCategory()</slot>
 </slots>
</ui>
